src/agent/user.go

```diff
@@ -0,0 +1,25 @@
+package agent
+
+
+// User represent a kitty cash user
+type User struct {
```

Comment:
I need everyone's opinion on this, but I was thinking about having email confirmations for the reservation process. So to initiate reservation, we get:
* What kitty to reserve.
* Payment method.
* Address to have kitty under.
* Captcha code.
* Send them confirmation code via email.
* Get them to input confirmation code.

Hence;
* Each email can only reserve "n" kitties at a time.
* But also, each address can only reserve "n" kitties at one time too.